python-simpleline.spec

```diff
@@ -10,7 +10,7 @@ Release: 1%{?dist}
 #   cd python-simpleline && make archive
 Source0: https://github.com/rhinstaller/python-%{srcname}/archive/%{srcname}-%{version}.tar.gz
 
-License: GPLv2+
+License: GPLv3+
```

Comment:
Shouldn't it be `LGPLv3+`?

Reply:
Haha, indeed. Maybe I should read commit messages :(

Reply:
Ups. Thanks a lot for spotting that!

Reply:
Fixed.